locksmithctl/daemon.go

```diff
@@ -116,6 +116,50 @@ func expBackoff(interval time.Duration) time.Duration {
 	return interval
 }
 
+func getPeriodicAndSleep() {
+	var period *timeutil.Periodic
+
+	startw := os.Getenv("LOCKSMITHD_REBOOT_WINDOW_START")
+	if startw == "" {
+		startw = os.Getenv("REBOOT_WINDOW_START")
+	}
+
+	lengthw := os.Getenv("LOCKSMITHD_REBOOT_WINDOW_LENGTH")
+	if lengthw == "" {
+		lengthw = os.Getenv("REBOOT_WINDOW_LENGTH")
+	}
+
+	if (startw == "") != (lengthw == "") {
+		dlog.Fatal("Either both or neither $REBOOT_WINDOW_START and $REBOOT_WINDOW_LENGTH must be set")
```

Comment:
This section is now a member of `getPeriodicAndSleep` function. It would be good to apply Go error handling pattern and log / raise errors in a _final_ step and not into the function:

```suggestion
		return errors.New("Either both or neither $REBOOT_WINDOW_STA...")
```

It would imply to change the method signature and to handle properly the error in the calls. :) 

ref:
- https://blog.golang.org/go1.13-errors
- https://blog.golang.org/error-handling-and-go